src/style.css

```diff
@@ -1,3 +1,77 @@
+:root {
+  --active-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: relative;
+  box-shadow: 0 2px 4px 0 #0000000d;
+  padding: 0 50px;
+}
+
+.nav-list {
+  list-style-type: none;
+  display: flex;
+  margin: 0;
+  padding-left: 0;
+}
+
+.nav-link {
+  display: flex;
+  position: relative;
+  align-items: center;
+  height: 60px;
+  margin-left: 20px;
+  color: black;
+  text-decoration: none;
+}
+
+.nav-item {
+  cursor: default;
+  justify-content: center;
+}
+
+.is-active {
+  margin-left: 0;
+  color: var(--active-color);
+}
+
+.nav__link:hover {
+  color: var(--active-color);
+}
```

Comment:
The selector `.nav__link:hover` does not match any class in the HTML file. It should be `.nav-link:hover` to correctly apply the hover effect to the navigation links.